docs/contracts/epic-planning-ref.md

```
@@ -0,0 +1,312 @@
+The commit has no merge parent and cannot include changes outside the candidate tree. Model output may supply a human summary, but that text is normalized and cannot change the closed identity trailers after `prepared`.
+
+## 9. CAS and verification
+
+`publish_artifact_pass` executes the following idempotent state machine.
+
+| Observation | Action |
+| --- | --- |
+| phase=`prepared`, object absent, ref=expected parent | write exact commit object; verify OID; record `commit_created` |
+| phase=`prepared`, expected object already exists | verify bytes/tree/parent/message; record `commit_created` |
+| phase=`commit_created`, ref=expected parent | CAS ref from parent to expected commit; record `ref_advanced` |
+| phase=`prepared|commit_created`, ref=expected commit | reconstruct a successful CAS receipt only after exact commit verification; record `ref_advanced` |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)